OKTA-413125 Overall style edit for OIE embedded auth content - part 1 (#2571)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/ios/redirecturi.md

>   If you're using the sample app, use `com.embeddedauth://callback`.

Reviewer: ```suggestion
  If you're using the sample app, use either `com.embeddedauth://callback` or
```

>   Otherwise, use a URI that your app accepts. Regardless of the value you use, the **Sign-in redirect URI** value must match the redirect URI configured in your app.

Reviewer: ```suggestion
a URI that is appropriate for your app. Regardless of the value that you use, the **Sign-in redirect URI** value must match the redirect URI that is configured in your app.
```
